pkg/commands/process/settings/rules/ruby/third_parties/airbrake.yml

```diff
@@ -0,0 +1,74 @@
+languages:
+  - ruby
+patterns:
+  - pattern: |
+      Airbrake.$<METHOD>($<...>$<DATA_TYPE>$<...>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+      - variable: METHOD
+        values:
+          - notify
+          - notify_sync
+          - notify_request
+          - notify_request_sync
+          - notify_query
+          - notify_query_sync
+          - notify_performance_breakdown
+          - notify_performance_breakdown_sync
+          - notify_queue
+          - notify_queue_sync
+  - pattern: |
+      Airbrake.$<METHOD>() do |$<NOTIFY:identifier>|
+        $<NOTIFY>[][] = $<DATA_TYPE>
+      end
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+      - variable: METHOD
+        values:
+          - notify
+          - notify_sync
+          - notify_request
+          - notify_request_sync
+          - notify_query
+          - notify_query_sync
+          - notify_performance_breakdown
+          - notify_performance_breakdown_sync
+          - notify_queue
+          - notify_queue_sync
```

Comment:
Not sure if we could use an auxiliary rule to avoid the repetition here. I chose not to because I think it keeps the rule easy to understand, but open to update it if folk feel strongly.